### ISSUE TITLE
License Clarification and PRs

### ISSUE THREAD

**Issue by tingold** [lukeroth/gdal#38]
First off, thanks for this repo! Works great right out of the box! Couple of things: 

1) Can you clarify the license a little? It grants permission to redistribute and extend with attribution but any thoughts on using a more standard MIT or Apache and then adding that clause? Just want to make sure I understand the intent.

2) There are some good PR's outstanding (specifically the VSI one) to which I've extended on my own branch but wondering if there is active dev going on here or should I fork the repo and continue dev there. 

3) Any thoughts about approaching OSGeo about integrating this into the GDAL baseline? SWIG seems  hopelessly broken for golang atm....

**Comment by lukeroth** [comment]
1.  That's a good point, when I put this repository together I didn't expect it to get much more than personal use.  I used a license file from another project that I thought was MIT but didn't notice was more restrictive.  I've updated the license to standard MIT.
2.  The repo has languished for a while as our company has not used Go extensively.  I've brought it up to speed with outstanding PRs, fixed issues with newer versions of GDAL, and merged from more active forks, but don't have the resources to bring this up to the level that I'd like.
3.  I agree about SWIG, after some time fighting with it I realized that writing manual wrappers would be much simpler, and the API for GDAL is fairly stable.  I don't know how interested the GDAL maintainers would be, they recently dropped the PHP and Ruby bindings so they seem to be going in the other direction.
